### DISCUSSION TITLE
Wizards in CxJs

### DISCUSSION THREAD

**Discussion by enoris75** [codaxy/cxjs#722] [General] [answered]
Hi,

I'm starting working on a Wizard-like UI integrated in our CxJs front-end.

React has a number of wizard related packages (e.g. **react-step-wizard**, **react-multistep**) which manages state, navigation and UI rendering of wizards (the latter with debatable levels of success, imho).

Does someone knows whether packages of these kind do exist fo CxJs (I couldn't find any).

Kind regards

E.

**Comment by mstijak** [comment]
I'm pretty sure there are none.

We implemented wizards in a few projects. Maybe we can provide an example. Are you looking for some specific functionality? 
Do you need to support many different wizards, or have just one which is really complicated?
This is an interesting area of improvement.

**Reply by enoris75** [reply]
As of now I'm thinking of a rather linear wizard (where in certain case a step or two could be skipped, depending on previous choices).

Any example would be massively apreciated.

**Comment by mstijak** [comment, marked as answer]
Here is a simple wizard that can be used as a starting point - https://codesandbox.io/s/bold-waterfall-3tl4u?file=/app/routes/default/wizard/index.js

Once you fill in the form, you can go to the next step...

**Reply by enoris75** [reply]
thank you very much!
